docs/synapse/userguides/ug011_storm_basics.rst

```diff
@@ -93,13 +64,158 @@ Working with Synapse data commonly involves three broad types of operations:
 Whether lifting, filtering, or pivoting across data in a Cortex, you need to be able to clearly specify the data you’re interested in – your selection criteria. In most cases, the criteria you specify will be based on one or more of the following:
 
 * A **property** (primary or secondary) on a node.
-* A **specific value** for a property (``<form>=<value>`` or ``<prop>=<value>``) on a node.
+* A **specific value** for a property (``<form>=<valu>`` or ``<prop>=<pval>``) on a node.
```

Comment:
I'm neutral - have been using <form>=<valu> for primary and <prop>=<pval> for secondary, so did the same here for consistency. Though I don't have strong feelings either way.